front-page.php

```diff
@@ -0,0 +1,24 @@
+<?php
+get_header();
+
+$splash_text = IGV_get_option('_igv_splash_text');
+$splash_text_en = IGV_get_option('_igv_splash_text_en');
+
```

Comment:
I'm pretty happy to leave how is really. It all goes down in the site options page and makes it less complicated for end user